### PR TITLE
``Tools/cases_generator``: Fix typos and incorrect comments.

### PR DESCRIPTION
**Pull request by Eclips4**
Continuation of #114844 and #114522
I hope this is the last change in this series 😄

### CODE REVIEW

File: Tools/cases_generator/py_metadata_generator.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
-"""Generate uop metedata.
+"""Generate opcode metadata.
```
Comment (gvanrossum) [inline Tools/cases_generator/py_metadata_generator.py]: Maybe add "for Python" to distinguish it from the previous file?

**Review by gvanrossum — APPROVED** [review]

File: Tools/cases_generator/py_metadata_generator.py (hunk 1 above)
Comment (Eclips4) [reply Tools/cases_generator/py_metadata_generator.py]: Good point. Done

File: Tools/cases_generator/opcode_metadata_generator.py [hunk 2]
```diff
@@ -1,6 +1,6 @@
-"""Generate uop metedata.
+"""Generate opcode metadata.
 Reads the instruction definitions from bytecodes.c.
-Writes the metadata to pycore_uop_metadata.h by default.
+Writes the metadata to pycore_opcode_metadata.h by default.
```
Comment (Eclips4) [inline Tools/cases_generator/opcode_metadata_generator.py:3]: May it worth to write the full path to file (applies to all other files(generators files))?

**Review by Fidget-Spinner — APPROVED** [review]

**Comment by gvanrossum** [comment]
Thanks again!

**Pull request merged**